Traceback (most recent call last):
  File "/usr/local/ftp_sync/bin/cookie_insert2.py", line 219, in <module>
    main()
  File "/usr/local/ftp_sync/bin/cookie_insert2.py", line 208, in main
    load_all_Standard(files_dir,cur,con, 1000)
  File "/usr/local/ftp_sync/bin/cookie_insert2.py", line 188, in load_all_Standard
    ret = csv_Standard(files_dir + f, ad_dict,cur,con,insert_interval)
  File "/usr/local/ftp_sync/bin/cookie_insert2.py", line 102, in csv_Standard
    stmt = insert_d[adID]["stmt"]
KeyError: 89647
